src/main/java/cms/gov/madie/measure/exceptions/InternalServerErrorException.java

```diff
@@ -0,0 +1,10 @@
+package cms.gov.madie.measure.exceptions;
+
+public class InternalServerErrorException extends RuntimeException {
```

Comment:
Would a built-in Exception be sufficient? I'm not seeing the benefit over using `RuntimeException` since this class doesn't appear to be adding any functionality.

Reply:
@jkotanchik-SB Thanks a lot Joe! I just made the change.

Reply:
Is the class necessary? Could the calling code use a builtin Exception type instead?